readme.md

```diff
@@ -82,7 +89,9 @@ var Panda = function(name, age) {
 
 Your Answer:
 ```text
-
+Panda.prototype.eat_bamboo: function(){
+  this.num_bamboo_eaten ++;
+}
```

Comment:
:tada: Nice! This works but usually no space between the incrementing variable and the operator